.github/workflows/ci2.yml

```diff
@@ -68,15 +68,15 @@ jobs:
           - distro: rocky
             base: centos.8
             with: rockylinux/rockylinux:8
-            java: yes
+            java: true
```

Comment:
With the PR we should be building Java for all builds so it no longer needs to be an matrix index, you can remove these lines.